OneSignalSDK/onesignal/src/main/java/com/onesignal/InAppMessageView.java

```diff
@@ -475,62 +485,83 @@ private void dereferenceViews() {
     }
 
     private void animateInAppMessage(WebViewManager.Position displayLocation, View messageView, View backgroundView) {
+        final CardView messageViewCardView = messageView.findViewWithTag(IN_APP_MESSAGE_CARD_VIEW_TAG);
+        Animation.AnimationListener cardViewAnimCallback = createAnimationListenerForAndroidApi23Elevation(messageViewCardView);
```

Comment:
shoudn't we set this only for Android 6? Since its the only case that we will listen for changes

Reply:
Ahh true good point, ill add a check for this

Reply:
Fixed!